src/config.js

```diff
@@ -21,15 +21,13 @@ export default {
 
   startOfWeek: 0, // 0 Sun 1 Mon....6 Sat
   minStay: 1, // can select one night
-  maxStay: 0, // can select infinite nights
-  monthStart: currDate.getUTCMonth(), // start with current month by default M '0...12'
-  yearStart: currDate.getUTCFullYear(), // start with current year YYYY
+  maxStay: Infinity, // can select infinite nights
+  monthStart: null, // start with current month by default M '0...12'
+  yearStart: null, // start with current year YYYY
```

Comment:
I'd probably have `maxStay: null` as default value, then in the code you can do `Number.isFinite(opts.maxStay)` to determine if something's been passed (isFinite will return true for all numbers, including `0` but excluding `NaN`, `Infinity` and `-Infinity`).

Reason is it's very strange to pass `Infinity` as an init option, or, rather, as any parameter. :smile: Or maybe it's just me who hasn't seen enough of it.

Reply:
@abel-n I get the point, IMO this is easier to read and keeps code shorter by removing unnecessary value checks  ` `

Reply:
I also see your point about the code having ~20 less characters, however, it's still very strange to pass `Infinity` to any function. It makes it a bit better that this is part of the default config, so ideally users would never actually pass `Infinity`. Notwithstanding, I'd have 20 more characters in code and have the cleanest possible "signature" of constructor / config. I'll let you decide which provides better ergonomics in the long run.